destination/destination.go

```diff
@@ -0,0 +1,115 @@
+// Copyright © 2022 Meroxa, Inc.
```

Comment:
just a quick note, this year should be 2023 for all the files, also, please make sure this header is included on the top of all the go files, that's why the linter is complaining

Reply:
omg thank you! I was going crazy on why its complaining 🙈